code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -123,45 +123,45 @@
 
 		H.shock_stage = max(H.shock_stage-(H.feral/20), 0) //if they lose enough health to hit softcrit, handle_shock() will keep resetting this. Otherwise, pissed off critters will lose shock faster than they gain it.
 
-		for(var/mob/living/M in viewers(H))
-			if(M != H) // someone in view
-				if(prob(0.5)) // 1 in 200 chance of doing something so as not to interrupt scenes
-					if(light_amount <= 0.5) // in the darkness
-						if(H.nutrition <= 250 || H.jitteriness > 0) //tell them that person looks like food and is good to pounce.
-							H << "<span class='info'> Secure in your hiding place, you still feel the urge to hunt, and [M] looks like an extremely tempting target...</span>"
-					else // in lit area
-						if(H.nutrition <= 250 || H.jitteriness > 0) //tell them that person looks like food. It CAN happen when you're not hungry enough to be feral, especially if coffee is involved.
-							H << "<span class='danger'> Every movement, every flick, every sight and sound has your full attention, your hunting instincts on high alert... In fact, [M] looks extremely appetizing...</span>"
-							if(H.stat == CONSCIOUS)
-								H.emote("twitch")
-							if(!H.handling_hal)
-								spawn H.handle_feral()
-
-			else if(M == H) //nobody is in view.
-				if(light_amount <= 0.5) // in the darkness
-					if(prob(1)) //periodic nagmessages
-						if(H.nutrition <= 100) //If hungry, nag them to go and find someone or something to eat.
-							H << "<span class='info'> Secure in your hiding place, your hunger still gnaws at you. You need to track down some food...</span>"
-						else if(H.jitteriness >= 100)
-							H << "<span class='info'> sneakysneakyyesyesyescleverhidingfindthingsyessssss</span>"
-						else //otherwise, just tell them to hide.
-							H << "<span class='info'> ...safe...</span>"
-				else // in light
+		if(light_amount <= 0.5) // in the darkness. No need for custom scene-protection checks as it's just an occational infomessage.
+			if(prob(2)) //periodic nagmessages just to remind 'em they're still feral
```

Comment:
Combine needlessly nested if(). Same on line 138 and 139.

Reply:
The if on line 126 has an else on line 136 that handles what happens if the chimera's in the light. Combining the ifs would make it fail the darkness check (and thus check for oviewers and cause hallucinations) even when dark if the prob() fails.

Likewise, the if on 138 has an else on 149 that doesn't depend on 139.

Reply:
Ahh, alright.